CMakeLists.txt

```diff
@@ -314,6 +318,25 @@ option(NO_DX "Disable DirectX support")
 option(NO_TESTS "Disable all tests")
 option(NO_GLTESTS "Disable GL tests")
 
+option(MSVC_STATIC_CRT_RELEASE "Statically link MSVC CRT in Release" OFF)
+option(MSVC_STATIC_CRT_DEBUG "Statically link MSVC CRT in Debug" OFF)
+
+if(MSVC AND MSVC_STATIC_CRT_RELEASE)
+    message(STATUS "Using static MSVC CRT Release")
+    # http://stackoverflow.com/a/32128977/486990
+    add_compile_options(
+        "$<$<CONFIG:RelWithDebInfo>:/MT>"
+        "$<$<CONFIG:Release>:/MT>"
+        "$<$<CONFIG:MinSizeRel>:/MT>"
+    )
+endif()
+if(MSVC AND MSVC_STATIC_CRT_DEBUG)
```

Comment:
Also not sure if it's desirable to have separate config for Release and Debug on this. The reason I did this was that in the current project I have I actually have dynamic CRT in Debug and static CRT in Release because I needed static only for released products. I have no strong attachments to this setup and I can simplify to make the flag affect both configs.